src/linear_algebra/free_module/finite/rank.lean

```diff
@@ -122,4 +122,15 @@ by { simp [finrank] }
 
 end comm_ring
 
+section field
+
+variables [field R]
+variables [add_comm_group M] [module R M] [module.free R M] [module.finite R M]
+variables [add_comm_group N] [module R N] [module.free R N] [module.finite R N]
+
+lemma finite_dimensional.tensor_product : finite_dimensional R (M ⊗[R] N) :=
+by simp only [finite_dimensional, module.finite.tensor_product]
```

Comment:
Presumably this is just 
```suggestion
module.finite.tensor_product
```

Reply:
It wasn't happy with just that I thought, but just checking again it appears happy with it after a few underscores... thanks.

Though the linter is now saying the `module.free` assumption isn't necessary -- I put this here because I found the `finrank_tensor_product` in it, but now I'm less sure where it belongs?